blazor/common/data-binding/dapper-using-webapp.md

```diff
@@ -0,0 +1,570 @@
+
+## Prerequisite software
+
+* Visual Studio 2022.
+* MS SQL Server.
+
+## Creating Blazor application
+
+* Open Visual Studio and follow the steps in the [documentation](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-8.0&pivots=windows) to create the Blazor Web App.
+
+## Creating the database
+
+First, create a database named `BugTracker` and a table named `Bugs` to hold the list of bugs.
+
+1. Open SQL Server 2017 / latest version.
+2. Now, create a new database named `BugTracker`.
+3. Right-click on the created database and select New Query.
+4. Use the following SQL query to create a table named Bugs.
+
+```
+Create Table Bugs(
+Id BigInt Identity(1,1) Primary Key Not Null,
+Summary Varchar(400) Not Null,
+BugPriority Varchar(100) Not Null,
+Assignee Varchar(100),
+BugStatus Varchar(100) Not Null)
+```
+
+Now, the table design will look like below.
+
+![Bug Table Design in Blazor](../images/bug-table-design.png)
+
+## Creating data access layer
+
+Before creating a data access layer, you need to include the following code snippet in the `appsettings.json` file of server side project of Blazor Web App .
+
+{% tabs %}
+{% highlight C# tabtitle="app settings.json" hl_lines="2 10 11 12" %}
+
+          // your localhost portal number
+"BaseUri": "https://localhost:7105",
+"Logging": {
+  "LogLevel": {
+    "Default": "Information",
+    "Microsoft.AspNetCore": "Warning"
+  }
+},
+"AllowedHosts": "*",
+"ConnectionStrings": {
+  "BugTrackerDatabase": "Server= {Your Server Name};Database=BugTracker;Integrated Security=True"
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Connection String in appsettings](../images/connection-strings.png)
+
+## Adding Dapper package and creating a model class
+
+If you have created a Blazor Web App with the `Interactive render mode` set to `WebAssembly` or `Auto` ensure to follow these steps:
+
+1. Create a new project with Class Library template named `BlazorWebApp.Shared` for the database accessing purpose.
+
+To use Dapper and access database in the application, you need to install the following `NuGet` packages in server side & Shared projects .
+
+Run the following commands in the Package Manager Console.
+
+* The following command enable us to use Dapper in our application.
+
+    ```
+    Install-Package Dapper -Version 2.1.24
+
+    ```
+
+* The following command provide database access classes such as  `SqlConnection`, `SqlCommand`, etc. Also provides data provider for MS SQL Server.
+
+    ```
+    Install-Package Microsoft.Data.SqlClient -Version 5.1.2
+    ```
+
+2. Most of the ORMs provide scaffolding options to create model classes. Dapper doesn’t have any in-built scaffolding option. So, you need to create model class manually. In `BlazorWebApp.Shared` project , you are creating a class named `Bug.cs` in the `Data` folder  as follows.
+
+```c#
+
+namespace {Your namespace}
```

Comment:
Few places, you have used "BlazorWebApp". Consistently use, the namespace.

Reply:
namespace updated consistently.